docs/Antora/modules/howto_guides/pages/howto_read_aligned_signals.adoc

```diff
@@ -260,6 +275,93 @@ Cpp::
 ----
 ====
 
+== Creating MutiReader with builder
+
+With the builder, developers can add signals and input ports using the methods `addSignal` and `addInputPort`. For signals, the builder creates an internal input port connected to the input signal.
+
+By default, the value type is set as `daq::SampleType::Float64` and the domain type as `daq::SampleType::Int64`. This can be overridden with the methods `setValueReadType` and setDomainReadType.
+
+The default value of the read mode is `daq::ReadMode::Scaled`, which can be configured using the method `setReadMode` to `daq::ReadMode::Unscaled` or `daq::ReadMode::RawValue`.
+
+In Multi-Reader, developers can set the read timeout type. The default value is `daq::ReadTimeoutType::All`, which waits for the requested amount or until the timeout is exceeded. It can be set as `daq::ReadTimeoutType::Any`, meaning the timeout will wait until any available data or the timeout is reached.
+
+The builder has fields for a common sample rate, which is disabled by default (set to -1), and for starting on the full unit of the domain (also disabled by default). These members can be overridden with the methods `setRequiredCommonSampleRate` and `setStartOnFullUnitOfDomain`.
+
+[#sync5]
+.Creating multi reader with default builder
```

Comment:
As mentioned in the daily, you should not create mock signals here. Just assume they exist and create a multi reader using the builder.